internal/blockchain/v0/pool.go

```diff
@@ -216,6 +223,14 @@ func (pool *BlockPool) PopRequest() {
 		delete(pool.requesters, pool.height)
 		pool.height++
 		pool.lastAdvance = time.Now()
+
+		// the lastSyncRate will be updated every 100 blocks, it uses the adaptive filter
+		// to smooth the block sync rate and the unit represents the number of blocks per second.
+		if pool.height%100 == 0 {
```

Comment:
```suggestion
		if (pool.height-pool.startHeight)%100 == 0 {
```
We might have an initial height that skewers this metric